recipes/centos_jdk8/contrib/settings.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
there is a missing license file

Reply:
will be sending another PR for child images deriving out of it, will apply these comments

Reply:
@kameshsampath @eivantsov provided https://github.com/eclipse/che-dockerfiles/pull/141